app/guid-node/metadata/-components/metadata-cedar/template.hbs

```diff
@@ -0,0 +1,17 @@
+<div local-class='metadata-cedar-container'>
+    <div local-class='cedar-container'>
+        <CedarMetadataEditor
+            @templateObject={{@template.template}}
+        />
+    </div>
+    <div local-class='return-container'>
+        <Button
+            data-test-select-button
+            aria-label= {{ t 'metadata.add-flow-select' }}
```

Comment:
You can remove this `aria-label` as this button already has visible text that screenreaders will automatically use. The aria-label here doesn't seem to match the actual text of the button either.